### DISCUSSION TITLE
stackviewについて

### DISCUSSION THREAD

**Discussion by atsujr** [lit-iphone-course/lit-iphone-course#70] [Question-original] [answered]
(1) 実現したいこと
tableviewのセルのデザインを、以下の画像のようにしたい
(2) 実現するために自分が試した内容とその結果
autolayoutの教科書を参考に、stackviewをいろいろとつけてみたが。画像の位置をうまく調整できない(具体的には、以下の画像のように横をつめつめじゃないようにしたい)
(3) 調べた内容や資料
https://qiita.com/kinopontas/items/d08f84dbb711c5acbe28
(4) 原因だと思われそうな箇所(ソースコードをコピペして貼る)
stackviewにつける制約が間違えている様子

**Comment by txxxxc** [comment]
画像貼れてないっぽいので貼ってもらえると嬉しい🙏

**Reply by atsujr** [reply]
![スクリーンショット 2022-09-28 15 21 42](https://user-images.githubusercontent.com/105122292/192703093-47d9121d-1f95-4b34-9dc6-192ace48f728.png)
ごめんなさい。これが１枚目で、
![IMG_8564](https://user-images.githubusercontent.com/105122292/192703174-64cc10d5-8e86-4a11-80ea-bc17e8851c43.PNG)
これが2枚目です！

**Comment by txxxxc** [comment, marked as answer]
ImageViewに直接制約をつけるのではなくて、親のStackViewの成約に対してconstantをつけるといい感じに調整できるのでやってみてくださいー！
![image](https://user-images.githubusercontent.com/22112440/192708133-71c25749-af49-4f0e-a671-316a4b33c4e9.png)
